okapi-core/src/test/java/org/folio/okapi/InstallTest.java

```diff
@@ -1092,17 +1096,22 @@ void deployAsyncInitModule(TestContext context, String module, int port) {
         c.getLastReport().isEmpty());
   }
 
-  JsonObject enableAndWait(TestContext context, String tenant, String module) {
+  JsonObject installAndWait(TestContext context, String tenant, String module) {
+    return installAndWait(context, tenant, module, "enable", "?async=true");
+  }
+
+  JsonObject installAndWait(TestContext context, String tenant, String module, String action, String installParameters) {
     RestAssuredClient c = api.createRestAssured3();
     Response r = c.given()
         .header("Content-Type", "application/json")
-        .body("[ {\"id\" : \"" + module + "\", \"action\" : \"enable\"} ]")
-        .post("/_/proxy/tenants/" + tenant + "/install?async=true")
+        .body(new JsonArray().add(new JsonObject()
+            .put("id", module)
+            .put("action", action)
+        ).encode())
+        .post("/_/proxy/tenants/" + tenant + "/install" + installParameters)
         .then().statusCode(201)
-        .body(equalTo("[ {" + LS
-            + "  \"id\" : \""+ module + "\"," + LS
-            + "  \"action\" : \"enable\"" + LS
-            + "} ]"))
+        .body("[0].id", is(module))
+        .body("[0].action", is(action))
         .extract().response();
```

Comment:
`.extract().header("Location");` avoids `Response r` variable.

Reply:
ok.. If you look for things in Okapi test that could be improved you'll find a lot and I could be spending quite some time on this ...